loadtests/loadtest/calls.py

```diff
@@ -24,16 +25,14 @@ def get_auth(self):
         self.account_server_url = os.getenv("FXA_URL", DEFAULT_FXA_URL)
         random_user = uuid.uuid4().hex
         self.user_email = "loop-%s@restmail.net" % random_user
-        acct = TestEmailAccount(self.user_email)
         client = Client(self.account_server_url)
-        fxa_session = client.create_account(self.user_email,
-                                            password=random_user)
-
-        def is_verify_email(m):
-            return "x-verify-code" in m["headers"]
-
-        message = acct.wait_for_email(is_verify_email)
-        fxa_session.verify_email_code(message["headers"]["x-verify-code"])
+        try:
+            client.create_account(self.user_email,
+                                  password=random_user,
+                                  preVerified=True)
```

Comment:
LGTM, but for future readers, it may be worth a comment to call out that this only works in specially-enabled environments like stage.